src/codeEditor/index.tsx

```diff
@@ -1,4 +1,5 @@
-import PropTypes from 'prop-types';
```

Comment:
Are we still using `prop-types` anywhere after these TS refactors?

Reply:
yeah it's still being used here:

https://github.com/readmeio/syntax-highlighter/blob/48be9e45c8b39847b70f16849d990def53d74b6d/src/codeMirror/index.jsx#L9

this component looked rather gnarly so i didn't bother touching it